spacecmd/src/spacecmd/utils.py

```diff
@@ -124,21 +124,16 @@ def is_interactive(options):
 
 def load_cache(cachefile):
     data = {}
+    cachefile = f"{cachefile}.json"
     expire = datetime.now()
 
     logging.debug('Loading cache from %s', cachefile)
 
     if os.path.isfile(cachefile):
         try:
-            inputfile = open(cachefile, 'rb')
-            data = pickle.load(inputfile)
-            inputfile.close()
-        except (EOFError, pickle.UnpicklingError) as exc:
-            # If cache generation is interrupted (e.g by ctrl-c) you can end up
-            # with an EOFError exception due to the partial picked file
-            # So we catch this error and remove the corrupt partial file
-            # If you don't do this then spacecmd will fail with an unhandled
-            # exception until the partial file is manually removed
+            with open(cachefile, 'r', encoding='utf-8') as f:
+                data = json.loads(f.read())
```

Comment:
[nitpick]: Is there a particular reason that use `loads(f.read())` instead of `load(f)`?